Rewrite the math chapter (#47)
oetiker/lshort: src/math.tex

These are useful if you want to, for example, break an equation into multiple
lines with alignment, but still retain only a single equation number.
\begin{example}
\begin{equation}
  \begin{aligned}
    2 + 2 \times 2 
      &= 2 \times 2 + 2 \\ 
      &\neq 2 \times (2 + 2) \\ 
  \end{aligned}
\end{equation}
\end{example}
They can be also freely nested, if a need arises.
\begin{example}
\begin{align}
  a + b + c
    &= \begin{multlined}[t]
      long \\
      multiline \\
      equation \\
      inside \\
      align
    \end{multlined} \\
    &= \begin{aligned}[t]
      1 &+ long \\
        &- aligned \\
        &+ equation \\
        &\div inside \\
        &\times align
    \end{aligned} \\
    &= d + e + f
\end{align}
\end{example}

\subsection{IEEEeqnarray Environment}\label{sec:IEEEeqnarray_intro}

If the environments defined in \ei{amsmath} do not meet your needs, you can try
the \ei{IEEEeqnarray} from the \pai*{IEEEtrantools} package. It allows you to
specify the alignment of each math column separately similarly to the
\env{tabular} environment. Here we will present some of its basic
functionalities. More can found in its official manual~\cite{IEEEtran_HOWTO}.

> The part about \texttt{IEEEeqnarray} can be found in Appendix~F.

Reviewer: aargh ... I understood that you were refering to a not yet written additional appendix of lshort :)

> functionalities. More can found in its official manual~\cite{IEEEtran_HOWTO}.
> The part about \texttt{IEEEeqnarray} can be found in Appendix~F.

Reviewer: ```suggestion
functionalities. More information about \texttt{IEEEeqnarray} can found in \cite{IEEEtran_HOWTO} Appendix~F.
```
